src/NuGet.Clients/NuGet.PackageManagement.UI/Options/PackageSourcesOptionsControl.resx

```diff
@@ -677,17 +680,20 @@
   <data name="tableLayoutPanel2.LayoutSettings" type="System.Windows.Forms.TableLayoutSettings, System.Windows.Forms">
     <value>&lt;?xml version="1.0" encoding="utf-16"?&gt;&lt;TableLayoutSettings&gt;&lt;Controls&gt;&lt;Control Name="HeaderLabel" Row="0" RowSpan="1" Column="0" ColumnSpan="1" /&gt;&lt;Control Name="addButton" Row="0" RowSpan="1" Column="1" ColumnSpan="1" /&gt;&lt;Control Name="removeButton" Row="0" RowSpan="1" Column="2" ColumnSpan="1" /&gt;&lt;Control Name="MoveUpButton" Row="0" RowSpan="1" Column="3" ColumnSpan="1" /&gt;&lt;Control Name="MoveDownButton" Row="0" RowSpan="1" Column="4" ColumnSpan="1" /&gt;&lt;/Controls&gt;&lt;Columns Styles="Percent,100,Absolute,33,Absolute,33,Absolute,33,Absolute,30" /&gt;&lt;Rows Styles="Percent,100" /&gt;&lt;/TableLayoutSettings&gt;</value>
   </data>
+  <data name="MachineWideSourcesLabel.AutoSize" type="System.Boolean, mscorlib">
+    <value>True</value>
+  </data>
   <data name="MachineWideSourcesLabel.ImeMode" type="System.Windows.Forms.ImeMode, System.Windows.Forms">
     <value>NoControl</value>
   </data>
   <data name="MachineWideSourcesLabel.Location" type="System.Drawing.Point, System.Drawing">
-    <value>0, 157</value>
+    <value>0, 133</value>
   </data>
   <data name="MachineWideSourcesLabel.Margin" type="System.Windows.Forms.Padding, System.Windows.Forms">
     <value>0, 0, 0, 0</value>
   </data>
   <data name="MachineWideSourcesLabel.Size" type="System.Drawing.Size, System.Drawing">
-    <value>166, 23</value>
+    <value>321, 25</value>
```

Comment:
Are the localization team able to see an example of the dialog/window for each language, and modify the value for each language?  If not, do you need to check every language to make sure the size is large enough to display the appropriate label?

Reply:
I didn't actually change the size values. There's no reason this should have been updated by me simply setting an AccessibleName on two lists. This was automatically output when I clicked save. I repeated the process two times to ensure I didn't accidentally introduce a size change and it produced the same result.

Reply:
To add to this, it seems WinForms stores all properties you set in the GUI in the resx. It doesn't seem to be used exclusively for localization.

Reply:
@zivkan this is relevant: https://stackoverflow.com/questions/5543159/resource-files-get-modified-when-the-form-is-opened-in-visual-studio

It seems just opening the designer in Visual Studio on a machine with different hardware / OS will have the affect that some layout properties can change.